lib/Importer/Caffe2.cpp

```diff
@@ -445,6 +449,42 @@ void caffe2ModelLoader::loadWeights(caffe2::NetDef &net) {
       continue;
     }
 
+    if (typeName == "UniformFill") {
+      /*
+       output: "fc/w"
+       name: ""
+       type: "UniformFill"
+       arg {
+         name: "max"
+         f: 0.25
+       }
+       arg {
+         name: "shape"
+         ints: 1
+         ints: 16
+       }
+       arg {
+         name: "min"
+         f: -0.25
+       }
+      */
+      const auto &name = op.output(0);
+      auto *T = new Tensor();
+      tensors_[name] = T;
+      auto dim = getShape(dict["shape"]);
+      T->reset(ElemKind::FloatTy, dim);
+      auto TH = T->getHandle<>();
+      float tensorMin = loadFloat(dict["min"]);
+      float tensorMax = loadFloat(dict["max"]);
+
+      // Uniformly generate numbers in [tensorMin; tensorMax], inclusive.
```

Comment:
Isn't this supposed to be random, with a uniform distribution? https://github.com/caffe2/caffe2/blob/master/caffe2/operators/filler_op.h#L124

Reply:
Yes. But can we simulate Caffe2's random generator? Results will be different anyway.

I thought that for the purpose of testing, having this code is as good as having our own random generator in here. And I prefer not to have any randomness, to help with maintenance.

Reply:
Our PseudoRNG is deterministically random so I don't think it'd cause maintenance issues.

Reply:
Their random generator, at least for the CPU, is just `std::uniform_real_distribution()`: https://github.com/caffe2/caffe2/blob/master/caffe2/utils/math_cpu.cc#L831